tsdb/tsm1/engine.go

```diff
@@ -877,12 +892,10 @@ func (e *Engine) compactCache() {
 			span, ctx := tracing.StartSpanFromContextWithOperationName(context.Background(), "Engine.compactCache <-t.C")
```

Comment:
FYI we do create a wrapping span here to provide context in the UI. This function is run as a long-lived goroutine, and it wouldn't help us to create spans for an hours-long trace, so this is the right place to create a new trace, and give it an explicit, easy to read name. We sample every one of these traces.

It wouldn't hurt to change the operation name to `compact cache`.